### ISSUE TITLE
Suggestion: Provide an (optional) way to track the post-modification value.

### ISSUE THREAD

**Issue by phryneas** [pgMemento/pgMemento#43]
Hi there!

It would be great if there were a way to let pgMemento not only save the value before a transaction, but also the value after a transaction - currently we're caluclating the value after by executing `restore_record`, but this is quite expensive for a lot of changes at once, so doing the storage <-> runtime tradeoff would be well worth it.

Thank you very much!

(Here's what we are currently doing)

https://github.com/mayflower/postgraphile-audit-plugin/blob/e070f88eb9fc7cb352341e9c7e049966fffbf5fb/Setup.sql#L18-L61

**Comment by FxKu** [comment]
Hi Lenz,
it's true that seeing the full change with old and new values requires to call the function you are using. My original idea was to not store things twice, but I do see value in providing a separate log column as an optional feature. As I've already changed the schema of log tables in the current master, it might be the right time to add it now, before doing the `0.7` release.
Expect a PR soon. I've already started implementing something and, fortunately, it was quite straight forward so far. Only some DDL cases left - like adding or altering columns.
